openmetadata-service/src/main/java/org/openmetadata/service/search/SearchRequest.java

```diff
@@ -87,6 +90,10 @@ public List<String> getIncludeSourceFields() {
     return includeSourceFields;
   }
 
+  public boolean getHierarchy() {
```

Comment:
why we need this method?

Reply:
we have added a query param in searchapi endpoint  (&getHierarchy=true) , to represent that param we needed to add this method in [SearchRequest.java]